app/src/Pages/Home/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  getPublicData,
+  publicDataRequest,
+  allOffices,
+  availableCampaigns,
+  selectedOffices,
+  selectedCampaigns,
+  selectedStartDate,
+  selectedEndDate,
+  filteredPublicData,
+  campaignsTable,
+  mapData,
+  aggregatedContributorTypes,
+  aggregatedDonationSize,
+  aggregatedContributionsByRegion,
+  donationSizeByDonationRange,
+  setSelectedOffices,
+  setSelectedCampaigns,
+  setSelectedStartDate,
+  setSelectedEndDate,
+  summaryData,
+} from '../../state/ducks/publicData';
+import { showModal } from '../../state/ducks/modal';
+import Home from './Home';
+
+class HomePage extends React.Component {
+  componentDidMount() {
+    const { fetchPublicData } = this.props;
+    fetchPublicData();
+  }
+
+  render() {
+    return <Home {...this.props} />;
+  }
+}
+
+HomePage.propTypes = {
+  fetchPublicData: PropTypes.func,
+  request: PropTypes.shape({
+    isLoading: PropTypes.bool,
+    error: PropTypes.string,
+    data: PropTypes.object,
```

Comment:
Again, NBD, but having an abstracted PropTypes shape that is the SOT for the structure of this data could be helpful for future developers. It would be imported and you'd put something like:
```
HomePage.propTypes = {
  request: PropTypesSOT.requestObject
}
```

Reply:
Ooh, like this, but again going to hold off on refactoring as we've got this pattern in lots of places.